addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -318,7 +318,8 @@ const UserValueWidget = publicWidget.Widget.extend({
             this.illustrationEl = document.createElement('i');
             this.illustrationEl.classList.add('fa', this.options.dataAttributes.icon);
         }
-        if (this.options.dataAttributes.reload) {
+        // Set no-preview = true by default (avoid uncaught promise when not set)
```

Comment:
You could add something like "When the reload attribute is present in the data attributes." Here, we assume that's always the case.